cabal.project

```diff
@@ -65,6 +65,9 @@ constraints:
 
 tests: True
 
+package concat-examples
```

Comment:
I wouldn't mind a comment here that explained that this only exists because of the corresponding `source-repository-package` clause above.